modules/releasing/pages/create-release-plan.adoc

```diff
@@ -48,7 +48,7 @@ spec:
 <.> Optional: An unstructured key used for providing data for the managed Pipeline.
 <.> Optional: Reference to the Pipeline to be executed by the release service.
 <.> Optional: The name of the service account to use in the Pipeline to gain elevated privileges. It's used only if you have defined the `pipelineRef` value.
-<.> Optional: The number of days a Release should be kept before being garbage collected. (default is 7 days)
+<.> Optional: The number of days a Release CR should be kept in the cluster before being automatically deleted by KubeArchive (default: 5 days, maximum: 30 days). Deleted Releases are archived and remain accessible via the KubeArchive API. This impacts Snapshot garbage collection, as Snapshots are protected from deletion while their associated Release exists.
```

Comment:
what makes the default releaseGracePeriodDays 5? And stops you from having it be over 30? I don't see anything in the webhook for it